worker/src/core/loaders/message-worker/index.ts

```diff
@@ -158,6 +164,8 @@ const finalize = tracer.wrap(
             campaignId,
             action: 'finalize',
           })
+          // for each campaign id, send email confirmation
+          sendFinalizedNotification(campaignId)
```

Comment:
I think here is where you should use `void`

Reply:
this function does return void

Reply:
oh I meant like the [void operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void)